com.ibm.streamsx.topology/opt/python/include/splpy_tuple.h

```diff
@@ -36,14 +48,56 @@ namespace streamsx {
 
   /**
    * Convert the SPL tuple that represents a Python object
-   * to a Python tuple that holds:
-   *  - pickled value as a memory view object
+   * to a Python tuple that holds as its first argument.
+   * One or two arguments in a Python tuple are passed to the function
+   * (which is a wrapper around the user function).
+   * 
+   *  Python object was passed by reference in the SPL tuple
```

Comment:
Nit, could you start the sentence with `if`? 

I had to read this a couple times to understand that this was a description of the structure of the tuple based on whether the SPL blob contained a reference or a pickled object.